.github/workflows/test-down-level.yml

```diff
@@ -25,20 +25,20 @@ jobs:
       matrix:
         vec: [
```

Comment:
This test will be more complicated, because it downloads previously created artifacts that still use the openssl name. But going forward, they will be named quictls.

Reply:
so, whats the solution here?  We can back out the change I guess until such time as the new artifacts get built.

Reply:
Likely what we will have to do is add a `prevTls` parameter to point it to the old artifact name. We can probably make it optional, and update the logic below to use `prevTls` if set, otherwise `tls`.

Reply:
done, I opted to just introduce an imgtls element to each vector in the matrix to avoid having to do github actions yaml conditionals, which always seem hard to read to me, and made the schannel elements a 1:1 mapping.